x/bank/keeper/hooks_test.go

```diff
@@ -20,8 +20,8 @@ var _ types.BankHooks = &MockBankHooksReceiver{}
 // BankHooks event hooks for bank (noalias)
 type MockBankHooksReceiver struct{}
 
-// Mock BeforeSend bank hook that doesn't allow the sending of exactly 100 coins of any denom.
-func (h *MockBankHooksReceiver) BeforeSend(ctx sdk.Context, from, to sdk.AccAddress, amount sdk.Coins) error {
+// Mock BlockBeforeSend bank hook that doesn't allow the sending of exactly 100 coins of any denom.
+func (h *MockBankHooksReceiver) BlockBeforeSend(ctx sdk.Context, from, to sdk.AccAddress, amount sdk.Coins) error {
```

Comment:
asking to learn: Why 100coin denom for `BlockBeforeSend` and 50coins for  `TrackBeforeSend` ?